### PR TITLE
3.0 / pt - Bookmarker tutorial

### PR DESCRIPTION
**Pull request by andtxr**

### CODE REVIEW

File: pt/tutorials-and-examples/bookmarks/intro.rst [hunk 1]
```diff
@@ -1,13 +1,360 @@
...
+marcação (bookmarker). Para começar, nós vamos instalar o CakePHP, criar
+nosso banco de dados, e usar as ferramentas que o CakePHP fornece para obter
+nossa aplicação de pé rápido.
+
+Aqui está o que você vai precisar:
+
+#. Um servidor de banco de dados. Nós vamos usar o servidor MySQL neste
+   tutorial. Você precisa saber o suficiente sobre SQL para criar um banco de
+   dados: O CakePHP vai tomar as rédeas a partir daí. Desde que nós estamos
+   usando o MySQL, também certifique-se de que você tem a extensão ``pdo_mysql`
```
Comment (markstory) [inline pt/tutorials-and-examples/bookmarks/intro.rst]: Missing a `.

**Comment by markstory** [comment]
I can't read Portuguese but the formatting and code look reasonable.

File: pt/tutorials-and-examples/bookmarks/intro.rst (hunk 1 above)
Comment (andtxr) [inline pt/tutorials-and-examples/bookmarks/intro.rst]: Thanks Mark.

**Comment by CauanCabral** [comment]
Hi @andtxr , I think "Favoritos" is a best translation of "Bookmarker".

Anyway, :+1:

**Comment by andtxr** [comment]
Since there are no official help channels for each language I have decided to keep some keywords like it original to help in some trackings like issues, doubts, etc... . I am trying to join PT translation through @CakePHP-BR. If you feel confortable, do not hesitate to push. Thanks @CauanCabral! :+1:

**Pull request merged**